fulp_modules/main_features/bloodsuckers/code/bloodsucker/bloodsucker_crypt.dm

```diff
@@ -602,6 +613,121 @@
 			return
 
 
+/obj/structure/bloodsucker/vassalrack/proc/shapeshift_victim(mob/living/user, mob/living/target)
+	var/datum/antagonist/bloodsucker/bloodsuckerdatum = user.mind.has_antag_datum(/datum/antagonist/bloodsucker)
+	/// To deal with Blood
+	var/mob/living/carbon/human/C = user
+	var/mob/living/carbon/human/H = target
+
+	/// Dead? No worries! Shift them into one of our creations!
+	if(H.stat == DEAD)
+		to_chat(user, "<span class='notice'>Do you wish to rebuild this body as something graeter? This will remove any restraints they might have, make them into a mute husk, and will cost 150 Blood!</span>")
```

Comment:
```suggestion
		to_chat(user, "<span class='notice'>Do you wish to rebuild this body as something greater? This will remove any restraints they might have, make them into a mute husk, and will cost 150 Blood!</span>")
```